server/src/operators/qdrant_operator.rs

```diff
@@ -288,7 +288,7 @@ pub async fn create_new_qdrant_point_query(
         .await
         .map_err(|err| {
             log::info!("Failed inserting chunk to qdrant {:?}", err);
```

Comment:
Two things:

```suggestion
            log::error!("Failed inserting chunk to qdrant {:?}", err);
```

Then, can we also add a `sentry::capture_message(` here?

Reply:
I'll do you one better, added it so that it captures all errors when returning a `ServiceError` ( another point over `DefaultError`